### ISSUE TITLE
Wrong version of libcurl installed as part of vagrant 1.8.7

### ISSUE THREAD

**Issue by alantech21** [mitchellh/vagrant#8036]
Please note that the Vagrant issue tracker is reserved for bug reports and
enhancements. For general usage questions, please use the Vagrant mailing list:
https://groups.google.com/forum/#!forum/vagrant-up. Thank you!

### Vagrant version
Run `vagrant -v` to show the version. If you are not running the latest version
of Vagrant, please upgrade before submitting an issue.
1.8.7
### Host operating system
This is the operating system that you run locally.
macOS 10.12.1

### Guest operating system
This is the operating system you run in the virtual machine.
none yet.

### Vagrantfile
```ruby
# Copy-paste your Vagrantfile here
```
not applicable

Please note, if you are using Homestead or a different Vagrantfile format, we
may be unable to assist with your issue. Try to reproduce the issue using a
vanilla Vagrantfile first.

### Debug output
Provide a link to a GitHub Gist containing the complete debug output:
https://www.vagrantup.com/docs/other/debugging.html. The debug output should
be very long. Do NOT paste the debug output in the issue, just paste the
link to the Gist.

### Expected behavior
What should have happened?
vagrant command should have executed.

### Actual behavior
What actually happened?
Error message returned, and Crash Reporter runs.
Error is as follows:
dyld: Library not loaded: @rpath/libcurl.4.dylib
  Referenced from: /opt/vagrant/embedded/bin/curl
  Reason: Incompatible library version: curl requires version 9.0.0 or later, but libcurl.4.dylib provides version 7.0.0
Abort trap: 6

Short version: vagrant does not work at all.

One possibility: if @rpath is not set, it could be trying to use the libcurl.4.dylib in /usr/lib, but that one is actually a newer version than the vagrant included version.

### Steps to reproduce
1. enter command 'vagrant' with or without any action or arguments
2.
3.

### References
Are there any other GitHub issues (open or closed) that should be linked here?
For example:
- GH-1234
- ...

**Comment by lowescott** [comment]
This issue is also affecting my OS X systems, which are running 10.11.6. Normal Vagrant usage doesn't return an error; it was only through some debugging that I found I was also affected.

The error message returned is:
dyld: Library not loaded: @rpath/libcurl.4.dylib
  Referenced from: /opt/vagrant/embedded/bin/./curl
  Reason: Incompatible library version: curl requires version 9.0.0 or later, but libcurl.4.dylib provides version 7.0.0
Trace/BPT trap: 5

As a workaround, I symlinked OS X's version of `curl` in the `/opt/vagrant/embedded/bin` directory, and now things seem to work.

**Comment by leehus** [comment]
I got same problem

` INFO subprocess: Starting process: ["/opt/vagrant/embedded/bin/curl", "-q", "--fail", "--location", "--max-redirs", "10", "--user-agent", "Vagrant/1.8.7 (+https://www.vagrantup.com; ruby2.2.5)", "--continue-at", "-", "--output", "/Users/username/.vagrant.d/tmp/boxb7c2b1abb8b632b03ab2fd8887faea66ef659f68", "https://storage.googleapis.com/stable.release.core-os.net/amd64-usr/current/coreos_production_vagrant_vmware_fusion.json"]
 INFO subprocess: Command in the installer. Specifying DYLD_LIBRARY_PATH...
DEBUG subprocess: Selecting on IO
DEBUG subprocess: stderr: dyld: Library not loaded: @rpath/libcurl.4.dylib
  Referenced from: /opt/vagrant/embedded/bin/curl
  Reason: Incompatible library version: curl requires version 9.0.0 or later, but libcurl.4.dylib provides version 7.0.0`

os:10.12.1
vagrant 1.8.7
ruby:2.3.0

**Comment by leehus** [comment]
[#7969](https://github.com/mitchellh/vagrant/issues/7969)
it worked

**Comment by chrisroberts** [comment]
Closing as dup of #7969
